sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -340,10 +340,6 @@ export class MessageSender extends LinkEntity {
   async send(data: SendableMessageInfo): Promise<void> {
     throwErrorIfConnectionClosed(this._context.namespace);
     try {
-      if (!data || (data && typeof data !== "object")) {
```

Comment:
Why don't we need this?

Reply:
Because a few lines later we call `SendableMessageInfo.toAmqpMessage` which does the same check plus a lot more